### ISSUE TITLE
How Long To Train COCO?

### ISSUE THREAD

**Issue by rhecto1** [amdegroot/ssd.pytorch#501]
Right now I am using the COCO dataset instead of Pascal VOC. COCO has a lot more images so I'm not sure how long this is supposed to take. I am using an Nvidia RTX 2070 Super GPU for training with a batch size of 8. All other parameters are left as default. After 2 and a half hours I am at iteration 31,000 with the loss typically being about 6.0. Is there a point when I can expect this to finish? I couldn't find in the code how many epochs this is supposed to run for so I'm not sure how many iterations to expect. Is it the case that I am expected to just stop the training at some point otherwise it will just continue training forever? I appreciate any advice.

**Comment by AtriSaxena** [comment]
In general, if I say, you should see the loss if the loss is not improving after 2-3 epochs max. Then you can stop the training and change some hyperparameters such as batch size, learning rate etc. Then, again continue training you might improve.
